chicago/bills.py

```diff
@@ -34,8 +34,12 @@ def session(self, action_date) :
         # 2015 Kill Bill https://chicago.legistar.com/LegislationDetail.aspx?ID=2321351&GUID=FBA81B7C-8A33-4D6F-92A7-242B537069B3
         elif action_date < localize(datetime.datetime(2015, 5, 6)) :
             return "2011"
-        else :
+        # The 2019 Kill Bill has not been introduced yet, so this date
```

Comment:
if the session isn't technically over until this bill occurs, is is correct to define the 2019 leg session as starting 5/19 in the init module?